### PR TITLE
Connected UI to python script using flask.

### PR DESCRIPTION
**Pull request by Kashish121**
1. Added `src/mainScanner.py`  - Python script including flask for connectivity.
2. Minor changes in directory structure to enable compatibility with flask.
3. Linting issue taken care of.
4. Added steps on how to run python script to render output in UI `src/templates/index.html`.

@ishika1727 Please take a look and help me with the UI for the output.
@vinitshahdeo Please review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,49 @@ With this port scanner, I just attempt to connect<sup>This is a form of "reconna
...
+python -m pip install --user virtualenv
+python -m virtualenv --help
+```
+
+[Click here](https://virtualenv.pypa.io/en/latest/installation.html) if pip installation fails.
+
+#### Create virtual environment
+
+3. Create a project folder and a venv folder within:
+```js
```
Comment (vinitshahdeo) [inline README.md:54]: @Kashish121 Same here. Python not JS.

File: README.md [hunk 2]
```diff
@@ -27,6 +27,49 @@ With this port scanner, I just attempt to connect<sup>This is a form of "reconna
...
+python -m virtualenv venv
+\Python27\Scripts\virtualenv.exe venv
+```
+#### Activate the virtual environment
+
+4. Activate venv,
+
+```js
```
Comment (vinitshahdeo) [inline README.md:62]: @Kashish121 ?

File: README.md [hunk 3]
```diff
@@ -27,6 +27,49 @@ With this port scanner, I just attempt to connect<sup>This is a form of "reconna
...
+venv\Scripts\activate
+```
+
+5. Run 'mainScanner.py,
+
+```js
```
Comment (vinitshahdeo) [inline README.md:68]: @Kashish121 Same as above.

File: src/mainScanner.py [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+    # Check what time the scan started
+    t1 = datetime.now()
+
+    # Getting port range values from config.json
+    try:
+        with open(get_absolute_path('../config.json')) as config_file:
+            config = json.load(config_file)
+            print get_absolute_path('../config.json')
+            # '../config.json'
```
Comment (vinitshahdeo) [inline src/mainScanner.py:60]: @Kashish121 Redundant comment.

**Comment by Kashish121** [comment]
@vinitshahdeo  made the required changes.

**Pull request merged**